wg-ml-experience/charter.md

```diff
@@ -0,0 +1,63 @@
+# WG ML Experience
+
+This charter adheres to the conventions, roles, and organisation management outlined in [wg-governance] for the Working Group "ML Experience".
+
+## Scope
+
+The ML Experience Working Group focuses on developing, maintaining, and improving tools and extensions that support data science and machine learning practitioners workflows within Kubeflow. The group is dedicated to delivering a high-level, seamless experience integrated with the IDE of choice across multiple Kubeflow components.
+ 
+### In scope
+ 
+#### Code, Binaries, and Other relevant assets
+
+1. Development of Kubeflow JupyterLab extensions that provide simple abstractions and UX to interact with the most common Kubeflow components (e.g., pipelines, hyperparameter tuning) and shorten the time to value for practitioners comfortable with Jupyter. These extensions will focus on the most used Kubeflow components, such as:
+    - Pipelines;
+    - Training Operator & Katib;
+    - Model Registry;
+    - Model Serving (KServe);
+    - Feast.
+
+2. Promote the reusability of UI components from other Kubeflow UIs into the IDE (e.g., rendering a pipeline graph inside the JupyterLab environment) by establishing a shared contract between the IDE WG and the wider Kubeflow community. 
+
+3. Develop a Python SDK to simplify operationalization across Kubeflow components and provide a “one-stop-shop” for practitioners who want easy access to Kubeflow services. The SDK also provides the groundwork for the IDE extension automation and workflows.
+    - Create a single installation and configuration layer for users interacting programmatically with the Kubeflow ecosystem via SDKs.
+    - The “common” SDK is not meant to replace individual components’ SDKs but rather to offer a unified access layer to simplify dependency management and shared configuration (like authorization).
+
+#### Guiding Principles
+
+- Synergy among Kubeflow Working Groups: Collaborate with other WG to promote reusability of UI components from other Kubeflow UIs to create a single UX between Jupyter IDE and Kubeflow Central Dashboard;
+- Collaboration with other open-source IDE projects (like Jupyter and VSCode) to promote the creation and reusability of open standards for AI/ML tools (protocols, communication exchange, file formats, etc.) and plugins. The aim of this group is to actively participate in the development of these standards to include Kubeflow in a broader ecosystem or interoperable tools. 
```

Comment:
What kind of collaboration we see with VSCode community ? I am not familiar with their OSS community structure.

Reply:
I briefly collaborated with them a few years ago. If we work closely with the Jupyter community to define this set of patterns, we should at least try to engage them.

Reply:
In the FUP PR, I just highlight Jupyter as concrete example.